packages/react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -116,6 +108,24 @@ export const getStyles = (props: IMessageBarStyleProps): IMessageBarStyles => {
     },
   ];
 
+  const highContrastBorderColor: { [key: string]: string } = {
+    [MessageBarType.error]: 'rgba(255, 0, 0)',
+    [MessageBarType.blocked]: 'rgba(255, 0, 0)',
+    [MessageBarType.success]: palette.greenLight,
+    [MessageBarType.warning]: palette.yellowLight,
```

Comment:
Are these changes from the `rgba` values to palette colors also to meet contrast requirements?

Reply:
yes. I, also added styles for HC White because those colors for dark theme have an incorrect contrast ratio